src/pages/ShareTextPage.qml

```diff
@@ -0,0 +1,48 @@
+/****************************************************************************
```

Comment:
This actually should go to sailfish-components-webview as will be eventually shared component.